pages/hub/apps/guides/debridge.mdx

```diff
@@ -0,0 +1,301 @@
+</Callout>
+
+```ts
+import { encodeFunctionData } from 'viem'
+
+type DeBridgeCreateTxResponse = {
+  orderId: Hex
+  estimation: {
+    srcChainTokenIn: {
+      address: Address
+      name: string
+      symbol: string
+      decimals: number
+      amount: string
+      approximateOperatingExpense: string
+      mutatedWithOperatingExpense: boolean
+    },
+    srcChainTokenOut: {
+      address: Address
+      name: string
+      symbol: string
+      decimals: number
+      amount: string
+      maxRefundAmount: string
+    }
+    dstChainTokenOut: {
+      address: Address
+      name: string
+      symbol: string
+      decimals: number
+      amount: string
+      recommendedAmount: string
+      withoutAdditionalTakerRewardsAmount: string
+      maxTheoreticalAmount: string
+    },
+    recommendedSlippage: number
+    costsDetails: [ string ]
+  }
+  tx: {
+    to: Address
+    data: Hex
+    value: bigint
+  }
+  order: {
+    approximateFulfillmentDelay: number
+  },
+  prependedOperatingExpenseCost: string
+  fixFee: string
+  userPoints: number
+  integratorPoints: number
+}
+
+const fromChainId = 42161 // Arbitrum for example
+const fromTokenAddress = '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' // USDT
+const referralCode = '' // your deBridge refferal code
+
+const selections = [{...}] 
+const rawAmount = '...'
+const rawDeadline = '...'
+const affiliate = '...'
+const minOdds = '...'
+const betData = 0x...
+
+const params = new URLSearchParams({
+  dstChainId: 137, // Polygon
+  srcChainOrderAuthorityAddress: bettorAddress as string,
+  prependOperatingExpenses: 'false',
+  srcChainId: String(fromChainId),
+  srcChainTokenIn: fromTokenAddress,
+  srcChainTokenInAmount: 'auto',
+  dstChainTokenOut: '0xc2132d05d31c914a87c6611c10748aeb04b58e8f', // USDT in Polygon
+  dstChainTokenOutAmount: String(rawAmount),
+  dstChainTokenOutRecipient: bettorAddress as string,
+  dstChainOrderAuthorityAddress: bettorAddress as string,
+  externalCall: JSON.stringify({
+    version: 'evm_1',
+    fields: {
+      to: lpAddress,
+      data: encodeFunctionData({
+        abi: lpAbi,
+        functionName: 'betFor',
+        args: [
+          bettorAddress,
+          coreAddress,
+          rawAmount,
+          rawDeadline,
+          {
+            affiliate,
+            minOdds,
+            data: betData,
+          },
+        ],
+      }),
+    },
+  }),
+  referralCode: String(referralCode), // your referral code
+})
+const deBridgeCreateTxResponse = await fetch(`https://api.dln.trade/v1.0/dln/order/create-tx?${params}`)
+const { 
+  orderId, // deBridge order id
+  estimation, // information about gas expense, etc.
+  tx, // transaction data
+  fixFee // deBridge fixed fee in native tokens
+}: DeBridgeCreateTxResponse = await deBridgeCreateTxResponse.json()
+```
+
+<Callout type="warning">
+  It is required to use the `betFor` function instead of the `bet` otherwise bettor couldn redeem his bet.
```

Comment:
It's necessary to use the `betFor` function rather than the `bet` function to ensure that the bettor can redeem their bet.

Reply:
f13e4127dec44bcbf66bbd1bc0b1418ef32f9a51